frontend/src/api/calls/AssignmentCalls.ts

```diff
@@ -1,9 +1,113 @@
-import { getAllEntitiesFromLinksUrl } from "./baseCalls";
-import { assignmentCollectionName, IAssignment } from "../entities/AssignmentEntity";
+import { AxiosConf, getAllEntitiesFromLinksUrl } from "./baseCalls";
+import { Assignment, assignmentCollectionName, IAssignment } from "../entities/AssignmentEntity";
+import axios from "axios";
+import { IStudent } from "../entities/StudentEntity";
+import { IUser } from "../entities/UserEntity";
+import apiPaths from "../../properties/apiPaths";
+
+export type IFullAssignment = { assignment: IAssignment; student: IStudent; assigner: IUser }[];
```

Comment:
This is a weird name. Maybe you mean
```suggestion
export type IFullAssignment = { assignment: IAssignment; student: IStudent; assigner: IUser };
```